.github/workflows/test-branch.yml

```diff
@@ -13,9 +13,9 @@ jobs:
     strategy:
       matrix:
         core:
-          - {name: 'WP latest', version: 'latest'}
-          - {name: 'WP trunk', version: 'WordPress/WordPress#master'}
-          - {name: 'WP minimum', version: 'WordPress/WordPress#4.6'}
+          - {name: 'WP latest', version: 'latest', hasBlocksEditor: true}
```

Comment:
Agree. The initial reason for splitting the workflows is to save the energy used to run multiple Core versions. Then if we have any issue with nightly or minimum versions, we can address it in the follow-up PR like this one. But because we don't run tests against the minimum and nightly for PRs, we don't know if the test pass or not for those environments. I merged them in [7c925cf](https://github.com/10up/simple-podcasting/pull/164/commits/7c925cf475c8b924f364d3cd66c0d3634310b7ca).